### PR TITLE
change zenhub to use redis publisher for metrics. Change http metricp…

### PR DESCRIPTION
**Pull request by jplouis**
…ublisher and metric request client to use http keepalive

### CODE REVIEW

**Comment by jplouis** [comment]
retest this

**Comment by jplouis** [comment]
retest this

File: Products/ZenHub/zenhub.py [hunk 1]
```diff
@@ -536,11 +539,9 @@ def getRRDStats(self):
         threshs = perfConf.getThresholdInstances(BuiltInDS.sourcetype)
         threshold_notifier = ThresholdNotifier(self.sendEvent, threshs)
 
-        self.log.info('Will post metrics to: %s', self.options.metrics_store_url)
-        metric_writer = metricWriter(
-            self.options.zauthUsername,
-            self.options.zauthPassword,
-            self.options.metrics_store_url)
+        self.log.info("metricwriter....")
```
Comment (iancmcc) [inline Products/ZenHub/zenhub.py]: DEBUGZ

File: Products/ZenHub/zenhub.py [hunk 2]
```diff
@@ -536,11 +539,9 @@ def getRRDStats(self):
...
+        metric_writer = metricWriter()
+        self.log.info("got metricwriter....")
```
Comment (iancmcc) [inline Products/ZenHub/zenhub.py]: ALSO

**Pull request merged**